diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/LineLength.kt

```diff
@@ -138,7 +156,30 @@ class LineLength(configRules: List<RulesConfig>) : DiktatRule(
                 else -> parent = parent.treeParent
             }
         } while (parent.treeParent != null)
-        return LongLineFixableCases.None
+        return None()
+    }
+
+    /**
+     * Analyzes the Binary expression and decides to go higher level with the analysis or analyze at this level
+     */
+    private fun isConditionToUpAnalysisBinExpression(parent: ASTNode, offset: Int): Boolean {
+        val condition1 = parent.treeParent.elementType in listOf(BINARY_EXPRESSION, PARENTHESIZED)
```

Comment:
It was just an example, the name of variables should be **meanable**
```suggestion
        val parentIsBinExprOrParenthesized = parent.treeParent.elementType in listOf(BINARY_EXPRESSION, PARENTHESIZED)
```

Reply:
Done